### ISSUE TITLE
error with compile_dm_users

### ISSUE THREAD

**Issue by a-zndr** [hfaran/slack-export-viewer#58]
Exported today cannot get this to work from zip

running 

`sudo slack-export-viewer -z /Users/<usr>/Downloads/<domain>\ Slack\ export\ Apr\ 2\ 2018.zip`

errors

```
Traceback (most recent call last):
  File "/usr/local/bin/slack-export-viewer", line 9, in <module>
    load_entry_point('slack-export-viewer==0.4.2', 'console_scripts', 'slack-export-viewer')()
  File "/Library/Python/2.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/Library/Python/2.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/Library/Python/2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Library/Python/2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/Library/Python/2.7/site-packages/slackviewer/main.py", line 76, in main
    configure_app(app, archive, debug)
  File "/Library/Python/2.7/site-packages/slackviewer/main.py", line 43, in configure_app
    dm_users = compile_dm_users(path, user_data, dm_data, empty_dms)
  File "/Library/Python/2.7/site-packages/slackviewer/archive.py", line 98, in compile_dm_users
    user2 = user_data[dm["members"][1]]
KeyError: u'U7E7Z0YTB'
```

**Comment by hfaran** [comment]
@tscritch this code was added by you. There are no comments indicating exactly what this function does and why and how. Could you shed some insight? (And add some comments for this).

**Comment by a-zndr** [comment]
Thanks for following up @hfaran

**Comment by tscritch** [comment]
Could be that the user doesn't exist. I'll take a look and comments as soon as I can. Sorry about that!

**Comment by hfaran** [comment]
Resolved by #60
